components/iostreams/tests/regressions/lost_output_2236.cpp

```diff
@@ -337,8 +329,7 @@ namespace server {
     {
         if (cid > cd->cid)
         {
-            HPX_TEST(
-                "change of owner during recovery not handled" == nullptr);
+            HPX_TEST("change of owner during recovery not handled" == nullptr);
```

Comment:
Should we maybe change this to `HPX_TEST_MSG(false, "change of owner during recovery not handled")`? (There's another similar one hiding 12 lines above.)

Reply:
Sure, I can apply this change.